ChangeLog.md

```diff
@@ -3,6 +3,15 @@
 ## Next
 * 
 
+## Version 1.16.0
+* Add download badges to README.
```

Comment:
```suggestion
```
Our customers don't care about this. The Changelog should be a summary that only includes the things our customers care about.